bedrock/careers/templates/careers/benefits.html

```diff
@@ -7,124 +7,156 @@
 {% extends "careers/base.html" %}
 
 {% block careers_content %}
-
-<!-- Hero -->
   <header class="c-careers-text-hero">
     <div class="mzp-l-content c-text-hero-wrapper mzp-t-content-md">
-      <div class="c-text-hero-title">
-        <h1>Benefits for real life</h1>
-      </div>
+      <h1 class="c-text-hero-title">Benefits for real life</h1>
+
       <div class="c-text-hero-body">
-        <p>We’re proud to offer our employees these outstanding benefits, which set us apart from other companies in our industry.</p>
+        <p>We’re proud to offer our employees these outstanding benefits, which
```

Comment:
Wrapping text at ~80 columns makes things easier to read. We don't do this consistently but I'm trying to be better about it.